version/version_test.go

```diff
@@ -0,0 +1,11 @@
+package version
+
+import (
+	"testing"
+)
+
+func BenchmarkGetHumanVersion(b *testing.B) {
+	for i := 0; i < b.N; i++ {
+		GetHumanVersion()
```

Comment:
We're a little surprised that, because there's no assignment here, that the compiler doesn't do some optimizations but looks good.